cdk/tooling_environment/stacks/grafana/lambda_functions_dashboard.template.json

```diff
@@ -78,12 +78,110 @@
         ]
       },
       "gridPos": {
-        "h": 14,
-        "w": 4,
+        "h": 3,
+        "w": 12,
         "x": 0,
         "y": 0
       },
       "id": 2,
+      "options": {
+        "colorMode": "value",
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "vertical",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "text": {
+          "titleSize": 18,
+          "valueSize": 40
+        },
+        "textMode": "value_and_name",
+        "wideLayout": false
+      },
+      "pluginVersion": "10.2.2",
+      "targets": [
+        {
+          "database": "<<DATABASE_NAME>>",
+          "datasource": "Amazon-Timestream",
+          "measure": "execution",
+          "rawQuery": "SELECT sum(execution)as \"Total executions\", sum(succeeded) as \"Succeeded executions\", sum(failed) as \"Failed executions\", sum(retry_attempts) as \"Retry attempts\"\r\nFROM(\r\n  SELECT 1 as execution\r\n,(sum(invocation) over (partition by lambda_function_request_id)) -1 as retry_attempts\r\n, min(failed) over (partition by lambda_function_request_id) as failed\r\n, max(succeeded) over (partition by lambda_function_request_id) as succeeded\r\nFROM $__database.$__table \r\n  where $__timeFilter and resource_name in (${resource_name:singlequote})\r\n     and monitored_environment in (${monitored_env:singlequote})\r\n  ) t\r\n",
```

Comment:
query for readability:
![image](https://github.com/user-attachments/assets/bf935a27-c2a9-4ec0-ae1e-bae3e175e3a1)